Source/PermissionTypes/Photos.swift

```diff
@@ -35,7 +35,12 @@ internal extension Permission {
         }
     }
     
-    func requestPhotos(callback: Callback) {
+    func requestPhotos(_ callback: Callback) {
+        guard let _ = Bundle.main.object(forInfoDictionaryKey: .requestedPhotoLibraryUsageDescription) else {
```

Comment:
Since when do we need to add an usage description in the Info.plist for the Photos ?